app/static/styles/constants.css

```diff
@@ -0,0 +1,87 @@
+:root {
+  --dark-green: #064635;
+  --light-green: #9bd7c0;
+  --cream: #fffcf9;
+  --gray: #a6a6a6;
+  --light-orange: #f0bb62;
+  --dark-orange: #d76514;
+
+  --violet: #83a3ff;
+
+  --main: var(--dark-green);
+  --secondary: var(--violet);
+  --accent: var(--light-green);
+  --bg-white: var(--white);
+
+  --white: #e8f3f5;
+  --black: #151616;
+
+  --fs-h1: 2.3rem;
+  --fs-h2: 2.1rem;
+  --fs-h3: 1.8rem;
+  --fs-h4: 1.5rem;
+  --fs-p: 1rem;
+}
+
+body{
+  font-family: 'Ubuntu', sans-serif;
+}
+
+h1 {
+  font-size: var(--fs-h1);
+  color: var(--black);
+}
+
+h2 {
+  font-size: var(--fs-h2);
+  color: var(--black);
+}
+
+h3 {
+  font-size: var(--fs-h3);
+  color: var(--black);
+}
+
+h4 {
+  font-size: var(--fs-h4);
+  color: var(--black);
+}
+
+p,
+a,
+input,
+textarea,
+button,
+b {
+  font-size: var(--fs-p);
+}
+
+@media (max-width: 900px) {
+  :root {
+    --fs-h1: 2rem;
+    --fs-h2: 1.8rem;
+    --fs-h3: 1.6rem;
+    --fs-h4: 1.2rem;
+    --fs-p: 0.96rem;
+  }
+}
+
+@media (max-width: 640px) {
+  :root {
+    --fs-h1: 1.8rem;
+    --fs-h2: 1.6rem;
+    --fs-h3: 1.5rem;
+    --fs-h4: 1.1rem;
+    --fs-p: 0.88rem;
+  }
+}
+
+@media (max-width: 480px) {
+  :root {
+    --fs-h1: 1.6rem;
+    --fs-h2: 1.4rem;
+    --fs-h3: 1.3rem;
+    --fs-h4: 1rem;
+    --fs-p: 0.8rem;
+  }
+}
```

Comment:
Do you think changing html tag font-size would be beneficial here?
Example:
```css
html {
	font-size: 62,5%; // 16px * 0.625 = 10px;
}

h1 {
	font-size: 1.8rem; // 10px * 1.8 = 18px;
}
```

```css
html {
	font-size:100%; // 16px
}

h1 {
	font-size: 1.5rem; // 16px * 1.5 = 24px;
}
```

Reply:
@HulekJakub dziękuję za review <3 Nie jestem pewna jak to się będzie zachowywać przy zmienianiu powiększenia ekranu, zrobię research na ten temat